tests/resources/test_teams.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+import os
+from contextlib import contextmanager
+
+from tests.conftest import JiraTestCase, allow_on_cloud
+
+
+@allow_on_cloud
+class TeamsTests(JiraTestCase):
+    def setUp(self):
+        JiraTestCase.setUp(self)
+        self.test_team_name = f"testTeamFor_{self.test_manager.project_a}"
+        self.test_team_type = "OPEN"
+        self.org_id = os.environ["CI_JIRA_ORG_ID"]
```

Comment:
I'm not sure how to get the `org_id` of the testing organisation :thinking:

One solution would be to implement the organisation API and to create a new one before running the tests ?

Reply:
I created a new PR (#1803) based on this one but also adding Organisations API so that I'm able to test the Team API.